symbols/arm9.yml

```diff
@@ -686,7 +686,23 @@ arm9:
           - 0x2052418
           - 0x2054A60
           - 0x20609E8
-        EU: 0x200D6BC
+        EU:
+          - 0x200D6BC
```

Comment:
`arm5find` is actually giving me 18 addresses (both for NA/EU), but there are only 16 here. Are some of the matches invalid?